src/main/java/car/service/RacingGameService.java

```diff
@@ -0,0 +1,41 @@
+package car.service;
+
+import car.domain.Car;
+import car.domain.Cars;
+import car.domain.Name;
+import car.domain.RacingGame;
+import car.domain.condition.RandomRacingGameStrategyImpl;
+import car.ui.GameResult;
+import car.ui.Winner;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class RacingGameService {
+
+    public GameResult<List<Winner>> start(String carNames, int moveCount) {
+        validateMoveCount(moveCount);
+        List<Car> participants = validateCarNames(carNames);
+        Cars cars = Cars.of(participants);
+        RacingGame racingGame = new RacingGame(new RandomRacingGameStrategyImpl(), cars);
+        return racingGame.play(moveCount);
+    }
+
+    private List<Car> validateCarNames(String carNames) {
+        String[] carNamesArray = carNames.split(",");
+        List<Car> cars = new ArrayList<>();
+
+        for (String carName : carNamesArray) {
+            Name name = new Name(carName);
+            cars.add(new Car(name));
+        }
+
+        return cars;
```

Comment:
```suggestion
private List<Car> validateCarNames(String carNames) {
    return Arrays.stream(carNames.split(","))
            .map(Name::new)
            .map(Car::new)
            .collect(Collectors.toList());
}
```

`stream` 을 활용한다면 이렇게도 줄여볼수 있지 않을까요?

저는 "`stream" 을 왜 사용해야할까?`"에 대해서 많은 자료를 찾아보고 생각해봤던적이 있어요. 
단지 `"코드가 간략해져서?"`, `"좀 더 우아해보여서?"`


이런것 보다는, 제가 생각했을때 하나를 뽑자면, 변수선언을 줄일수 있는점이 좋아 보였어요.
변수 선언이 많아지면, 자연스럽게 코드가 길어지고, 개념들이 꼬이기 시작해요.
정말 간단한 코드도 어느순간 필요없는 개념들이 주입되기 시작하죠.


여러 장당점이 분명히 존재해요!`stream`을 활용에 대해 고민해보시면 좋을것 같아요!